example/build_3d_map.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+import argparse
+import sys
+import timeit
+
+import click
+import numpy as np
+import open3d
```

Comment:
@cpaxton - import open3d as o3d can fail on some older CPUs. It's worth making this import optional (similar to cv2 imports in habitat-lab: https://github.com/facebookresearch/habitat-lab/blob/cc6c8065d1018b45f19f56a84384e4407542a2cb/habitat-lab/habitat/core/utils.py#L72-L91).

Reply:
not possible in this script unfortunately, i'm using o3d for voxelization

this won't be critical path though